Explorer/Assets/DCL/AvatarRendering/Wearables/Helpers/WearableComponentsUtils.cs

```diff
@@ -32,6 +41,8 @@ public static class WearableComponentsUtils
 
         private static readonly IExtendedObjectPool<URLBuilder> URL_BUILDER_POOL = new ExtendedObjectPool<URLBuilder>(() => new URLBuilder(), defaultCapacity:2);
 
+        public static IWebRequestController WebRequestController = IWebRequestController.DEFAULT;
```

Comment:
We should not hardcode static instances, instead pass it as parameter

Reply:
Removed! Actaully this was old code